src/device/spu/noise.cpp

```diff
@@ -3,40 +3,40 @@
 
 namespace {
 std::array<int8_t, 64> noise_addition = {
-	{1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
-	 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
-	 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1,
-	 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1}};
+    {1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
```

Comment:
You can use clang-format to take care of these things.

Put [this](https://gist.github.com/JaCzekanski/b13ae5d839dfc2fd3029d21e2cd5879b) in .git/hooks/pre-commit to do it when commiting changes.

Reply:
Thanks for the tip.